employees_info/src/main/java/com/teachmeskills/servlet/UserDeleteServlet.java

```diff
@@ -0,0 +1,45 @@
+package com.teachmeskills.servlet;
+
+import com.teachmeskills.util.PostgresDriverManager;
+import jakarta.servlet.ServletException;
+import jakarta.servlet.annotation.WebServlet;
+import jakarta.servlet.http.HttpServlet;
+import jakarta.servlet.http.HttpServletRequest;
+import jakarta.servlet.http.HttpServletResponse;
+
+import java.io.IOException;
+import java.sql.Connection;
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+
+@WebServlet("/delete")
+public class UserDeleteServlet extends HttpServlet {
+    @Override
+    protected void doGet(HttpServletRequest req, HttpServletResponse resp) throws ServletException, IOException {
+        try {
+        PostgresDriverManager driverManager = PostgresDriverManager.getInstance();
+        Connection connection = driverManager.getConnection();
+        PreparedStatement preparedStatement;
+        ResultSet resultSet;
+        preparedStatement = connection.prepareStatement("SELECT * FROM employees WHERE employee_id = ?");
+        preparedStatement.setInt(1, Integer.parseInt(req.getParameter("id")));
+        resultSet = preparedStatement.executeQuery();
+        if (!resultSet.isBeforeFirst()) {
+            req.setAttribute("status", "FALSE");

```

Comment:
Why do we need it?
it seems, if the deletion is not success, `executeQuery` method shoul return something

Reply:
My mistake, trying to check, have we smthng in result set or not. Are we need to check is resultSet.next == null?

Reply:
I don't think so, but you should check, which solution is appropriate

Reply:
Fixed.